src/freenet/l10n/freenet.l10n.es.properties

```diff
@@ -1384,10 +1485,13 @@ PluginManager.pluginReqNewerJVM=El complemento ${name} parece que requiere una v
 PluginManager.pluginReqNewerJVMTitle=El complemento ${name} necesita una versión de Java más reciente
 PluginManager.pluginLoadingFailedTitle=¡No se pudo cargar el complemento!
 PluginManager.pluginLoadingFailedShort=¡No se pudo cargar el complemento ${name}!
-PluginManager.pluginLoadingFailed=No se pudo cargar el complemento ${name} 
+PluginManager.pluginLoadingFailed=No se pudo cargar el complemento ${name}
 PluginManager.pluginLoadingFailedStillTryingOverFreenet=Aún estamos intentando cargar el complemento desde Freenet.
 PluginManager.pluginLoadingFailedWithMessage=El complemento ${name} no se pudo cargar: ${message}
+PluginManager.officialPluginLoadFailedSuggestTryAgainFreenet=Puede intentarlo de nuevo a través de Freenet, o puede obtenerlo desde la web.
+PluginManager.officialPluginLoadFailedSuggestTryAgainHTTPS=Puede obtener el complemento (plugin) a través de Freenet, o puede intentarlo de nuevo desde la web.
 PluginManager.officialPluginLoadFailedTryAgainFreenet=Cargar el complemento desde Freenet
+PluginManager.officialPluginLoadFailedTryAgain=Cargar el complemento vía web desde nuestros servidores (¡NO ES ANÓNIMO!)
```

Comment:
Hmm, does Transifex not automatically remove keys for which the source translation does not have a key anymore?